### ISSUE TITLE
run tests in isolated environment

### ISSUE THREAD

**Issue by lorenzofox3** [lorenzofox3/zora-dev-server#5]
In some cases one will want to run tests in an isolated environment such an iframe. Although it could be made at automation system level (puppeteer), having an endpoint right in the dev server could be nice

**Comment by munrocket** [comment]
Need this on another project :|

**Comment by lorenzofox3** [comment]
yes it is in the pipe. Meanwhile you can create you isolated environment with puppeteer frames. ZDS will later handle log aggregation etc
